test/libsolidity/smtCheckerTests/loops/for_loop_bmc_iterations_10.sol

```diff
@@ -0,0 +1,18 @@
+contract C
+{
+	uint256[] y;
+
+	function f() public view {
+		uint256 x = 0;
+		for (uint i = 0; i < y.length; i++) {
+			x = 1;
+		}
+		require(x != 0);
```

Comment:
What is the point of having a `require` at the end of the method? What is this testing actually?